### PR TITLE
New TAT ammo, and three AGLS changes

### PR DESCRIPTION
**Pull request by maikilangiolo**
## About The Pull Request
Added Beehive and Napalm AT36 ammo. They work exactly like GL54 airbust and incendiary, but scaled up for 37mm usage. They have the mech airbust slowdown and the GL54 1 tile knockback on direct hit (consistent with the other airbust shells, and makes sense with the moderate velocity of the ammo). Both new rounds have great FF potential, and are on par with other shells when comparing effectiveness.

https://user-images.githubusercontent.com/40152611/216758451-cf8207d9-4343-4b63-b35d-5c867b57157d.mp4

AGLS changes included are doubledd reload time (it was too fast), HP nerf (500 hp is TAT levels of HP), and armor buff (20 acid to bring it in line with the others, 100 bomb to avoid self nuking, especially because the ammo randomly drops in front of you). As well as a bug fix with turf targeting for AGLS frag ammo.
## Why It's Good For The Game
More ammo variation for the gun that arguably should enjoy juggling different ammo types for different targets and situations. And, no more fucked stats for the AGLS.
## Changelog
:cl:
add: added Napalm and Beehive AT36 shells
balance: AGLS reload 2 seconds --> 4 seconds
balance: AGLS HP 500 --> 300, bomb armor 75 --> 100, acid armor 0 --> 20
fix: fixed AT36 HE ammo being described as "high velocity" when its actually very slow
/:cl:

### CODE REVIEW

**Review by Lumipharon — CHANGES_REQUESTED** [review]
Don't make it 100 bomb armour. FF immunity is cringe.

**Comment by Lumipharon** [comment]
You should put a video on the pr btw

**Comment by maikilangiolo** [comment]
> You should put a video on the pr btw

videdo added

File: code/modules/projectiles/guns/mounted.dm [hunk 1]
```diff
@@ -509,6 +511,9 @@
 	undeploy_time = 3 SECONDS
 	deployable_item = /obj/machinery/deployable/mounted
 
-	max_integrity = 500
-	soft_armor = list(MELEE = 0, BULLET = 50, LASER = 0, ENERGY = 0, BOMB = 50, BIO = 100, FIRE = 0, ACID = 0)
+
+	max_integrity = 300
+	soft_armor = list("melee" = 0, "bullet" = 50, "laser" = 0, "energy" = 0, "bomb" = 80, "bio" = 100, "fire" = 0, "acid" = 20)
```
Comment (Lumipharon) [inline code/modules/projectiles/guns/mounted.dm:516]: defines.

**Review by Lumipharon — CHANGES_REQUESTED** [review]

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**